### ISSUE TITLE
diskid v.s. disk device name

### ISSUE THREAD

**Issue by dirkx** [Emrion/uploaders#4]
Currently the script relies on 

     sysctl -n kern.disks

to get the list of disks. May be useful to use

      gpart show | grep GPT | cut -wf 4

to get that list. As for example; on a plain 13.4 system I get:

      $  sysctl -n kern.disks
      ada5 ada4 ada6 ada3 ada1 ada2 ada0
      $ gpart show ada5
      gpart: No such geom: ada5.
      $ gpart show | grep GPT
      =>        34  7814037101  diskid/DISK-PL133181278121  GPT  (3.6T)

with that change - works nicely

**Comment by Emrion** [comment]
Thanks. I'm far from all PC for several days, therefore I can't investigate deeply this issue.
Nevertheless, i have two comments:
- Considering my code, this problem does not stop the script. It just doesn't examine the disk with such an error, right?
-  gpart show can lead to an error on hardened system with a non-root user. This is why I don't use that to list the disks.

This problem concerns only ada5 or all your disks? And, anyway, do you know why?

**Comment by Emrion** [comment]
I worked on this problem but I can't reproduce the fact that you have only the diskid as geom names. I played with kern.geom.label.disk_ident.enable and the like, but I still have ada like names. Can you help me to reproduce what you get?

**Comment by Emrion** [comment]
As you didn't answer, I did it on my own. 'gpart show' was the first means I used in this script to list the disks, but I found that some hardened systems do not allow this command (I mean without a geom as argument). Furthermore, this command lists also the virtual disks like zvol, not to speak about the duplicates between regular disk geom and diskid. So, it's more annoying than useful. I'll soon add the possibility to use 'gpart show' with the switch -g. Issue fixed.
